static/scss/answers/collapsible-filters/sdk-filter-overrides.scss

```diff
@@ -3,9 +3,11 @@
  $container-width-at-collapsed-filters-breakpoint: $breakpoint-collapsed-filters - (2 * $filters-width);
 
 @mixin CollapsibleFiltersOverrides {
-  .Answers-container {
-    @media (min-width: #{$breakpoint-expanded-filters}) {
-      width: math.div($container-width-at-collapsed-filters-breakpoint, $breakpoint-collapsed-filters) * 100%;
+  .Answers {
+    &-container, &-resultsMidContainer {
+      @media (min-width: #{$breakpoint-expanded-filters}) {
+        width: $container-width-at-collapsed-filters-breakpoint / $breakpoint-collapsed-filters * 100%;
```

Comment:
can we use math.div instead so that we don't get a webpack warning about the slash?